samples/notebook_instance/README.md

```diff
@@ -0,0 +1,47 @@
+# Notebook Instance Sample
+
+## Prerequisites
+
+This sample assumes that you have completed the [common prerequisites](/samples/README.md).
+
+### Update the Notebook Instance Specification
+
+Edit the roleARN value in my-notebook-instance.yaml to include the Sagemaker Execution permissions.
+
+## Using the Notebook Instance operator
+
+### Create a Notebook Instance
+
+This command creates a Sagemaker notebook instance based on the specification provided in my-notebook-instance.yaml.
+The Notebook Instance will start at the Pending state and will transition into InService once ready.
+
+```
```

Comment:
In all the other samples they contain a string saying something like `trainingjob.sagemaker.services.k8s.aws.amazon.com/my-training-job created` as an example https://github.com/aws-controllers-k8s/sagemaker-controller/tree/main/samples/training. Can you add that to stay consistent

Reply:
done